README.md

```diff
@@ -1,36 +1,339 @@
-This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
+![GO logo](./public/icons/logo-white-readme.svg)
 
-## Getting Started
+<p>
+  <br/>
+  Website for the public library that uses <a href="https://nextjs.org" target="_blank">Next.js</a> with the <a href="https://nextjs.org/docs/app" target="_blank">App Router</a> for the frontend and <a href="https://www.drupal.org/" target="_blank">Drupal</a> for content management.
+  <br>
+  Drupal CMS can be accessed through Lagoon by generating a one-time login in the preferred environment.
+  <br>
+  <br>
+  Last updated: 27/11/2024
+</p>
 
-First, run the development server:
+## URLs
+
+| Description                                             | URL                                                      |
+| ------------------------------------------------------- | -------------------------------------------------------- |
+| Demo site (may change)                                  | https://node.pr-1707.dpl-cms.dplplat01.dpl.reload.dk/    |
+| Demo site Drupal CMS (may change, login through lagoon) | https://varnish.pr-1707.dpl-cms.dplplat01.dpl.reload.dk/ |
+
+## Table of contents
+
+- [Getting started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [Technical Overview](#technical-overview)
+  - [Project structure](#project-structure)
+  - [git workflows](#git-workflows)
+- [Development](#development)
+  - [UI components from shadcn/ui](#ui-components-from-shadcnui)
+  - [Tailwind](#tailwind)
+  - [Codegen](#codegen)
+  - [Custom types](#custom-types)
+- [Deployment](#deployment)
+  - [git branches and pull requests](#git-branches-and-pull-requests)
+  - [Create pull request](#create-pull-request)
+  - [Reviewing a PR](#reviewing-a-pr)
+  - [Updating the demo site](#updating-the-demo-site)
+- [Quality assurance](#quality-assurance)
+  - [GitHub Workflows for quality assurance](#github-workflows-for-quality-assurance)
+- [Developers](#developers)
+
+## Getting started
+
+### Prerequisites
+
+- Use the node version registered in the [`.nvm`] file
+- Preferably managed by [`nvm`](https://github.com/nvm-sh/nvm)
+- This project uses [`yarn`](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable) make sure to have this installed globally on your machine
+
+### Setup
+
+1. Make sure you are using the correct Node version:
+
+```bash
+nvm use
+```
+
+2. Install dependencies:
+
+```bash
+yarn
+```
+
+3. Start the development server:
+
+We run the server in an [experimental https](https://nextjs.org/docs/pages/api-reference/cli/next#next-dev-options) state to not get blocked by CORS policy when developing locally.
+
```

Comment:
Somwhere here I would add that the dpl-cms needs to be running in order to get this going. 
Either through dapple or just `task dev:reset` in dpl-cms.